### PR TITLE
Adding nfv ovs dpdk & sriov scenario's.

### PR DESCRIPTION
**Pull request by Jaganathancse**
This changes to add NFV ovs dpdk and sriov reprodicers scenario's.

As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running
- [x] Appropriate documentation exists and/or is up-to-date:
  - [x] README in the role
  - [x] Content of the docs/source is reflecting the changes

### CODE REVIEW

File: scenarios/reproducers/va-nfv-ovs-dpdk-sriov.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 cifmw_architecture_scenario: "ovs-dpdk-sriov"
-
+cifmw_ci_gen_kustomize_values_ctlplane_interface: eno2
```
Comment (pablintino) [inline scenarios/reproducers/va-nfv-ovs-dpdk-sriov.yml:3]: Isn't this an environment detail that is more suitable of being stored DS?

**Comment by lewisdenny** [comment]
Hey @Jaganathancse is this patch still required?

**Comment by Jaganathancse** [comment]
> Hey @Jaganathancse is this patch still required?

Yes, need to update this PR.

**Comment by lewisdenny** [comment]
> > Hey @Jaganathancse is this patch still required?
> 
> Yes, need to update this PR.

Ack, setting it back to draft for now, please remove draft status when you would like to get the patch merged

**Comment by lewisdenny** [comment]
Hi @Jaganathancse 

Do you have any downstream tests ran yet for these new scenarios?

**Comment by pablintino** [comment]
Stale PR, closing. Feel free to reopen it if needed.

**Pull request closed without merging**